src/test/regress/expected/multi_create_table.out

```diff
@@ -518,3 +512,97 @@ SELECT create_distributed_table('orders_hash_part', 'o_orderkey');
  
 (1 row)
 
+-- Test rollback of create table
+BEGIN;
+CREATE TABLE rollback_table(id int, name varchar(20));
+SELECT create_distributed_table('rollback_table','id');
+ create_distributed_table 
+--------------------------
+ 
+(1 row)
+
+ROLLBACK;
+-- Table should not exist on the worker node
+\c - - - :worker_1_port
+\d rollback_table*
+\c - - - :master_port
+-- Again there should be no table on the worker node
+BEGIN;
+SELECT create_distributed_table('rollback_table','id');
+ERROR:  relation "rollback_table" does not exist
```

Comment:
Did you mean to do another `CREATE TABLE rollback_table(id int, name varchar(20));`?